src/build-2024/04 - Add Logging/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+
+var openAIChatCompletionModelName = "gpt-3.5-turbo"; // this could be other models like "gpt-4o".
+
+var builder = Kernel.CreateBuilder();
+
+// Add logging services to the builder
+builder.Services.AddLogging(b => b.AddConsole().SetMinimumLevel(LogLevel.Trace));
```

Comment:
Each of these projects can run independently and do not need to be learned in sequence. Since the code is not essential for running the project, I don't believe it should be shared across projects.